src/qt/veil/miningwidget.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019 The Veil developers
```

Comment:
NIT: Be sure to give appropriate copyright when adding files (e.g. 2021) or update to include 2021 if it's not done yet (there's a script we can run but might as well do it when we've edited a file)